less/navigation.less

```diff
@@ -0,0 +1,74 @@
+header {
+    width: 100%;
+    height: 70px;
+    background-color: #FFFFFF;
+    box-shadow: 2px gray;
+
+    @media @mobile {
+        height: 50px;
+    }
+}
+
+header nav {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  padding-top: 17px;
+  padding-bottom: 18px;
+  border-bottom: 1px solid #F7F7F7;
+
+  @media @mobile {
+      padding-top: 11px;
+      padding-bottom: 11px;
+  }
+}
+
+header nav:first-child {
+    padding-left: 32px;
+
+    @media @mobile {
+        padding-left: 24px;
+    }
+}
+
+header nav:last-child {
```

Comment:
Since you're referencing the same `header nav` tag here as you did twice above, I would recommend using the `&` pseudo-selector to put all of this in one block. For example ```header nav {
styles
styles
&:first-child {
styles
}
&:last-child {
styles
}
}```